.gitignore

```diff
@@ -122,6 +122,7 @@ db.sqlite3
 # Squire-specific gitignore (Secret Keys, etc.)
 /squire/secret_key.txt
 htdocs/media/images/achievements
+/components
```

Comment:
Q: What's in the components folder that it should be ignored?

Reply:
A folder that was left behind as a result of experimentation with different libraries. Seems like I accidentally pushed the modified .gitignore!